Orange/canvas/application/addons.py

```diff
@@ -150,6 +150,32 @@ def editorEvent(self, event, model, option, index):
         return model.setData(index, checkstate, Qt.CheckStateRole)
 
 
+def get_meta_from_archive(path):
+    """Return project name, version and summary extracted from
```

Comment:
If there is some way to do this more elegantly using _pkg_resources_ or _pip_, I'd sure like to know. :unamused: